### DISCUSSION TITLE
Enable AWS CloudWatch Namespace for IoT SiteWise

### DISCUSSION THREAD

**Discussion by mbeheralyell** [grafana/grafana#60001] [Data Sources]
Amazon Managed Grafana can be used for monitoring **data** from AWS SiteWise as it's available as one of the sources in the Grafana visualization panel. However, to monitor the health of the AWS IoTSiteWise components one needs to be able to monitor the AWS CloudWatch Metrics related to AWS IoT SiteWise Namespace e.g. Gateway.Heartbeat, OPCUAColector.Heartbeat, OPCUACollector.IncomingValuesCount, OPCUACollector.ActiveDataStreamCount, Gateway.PublishSuccessCount, etc. These dashboards can be created in AWS CloudWatch but not out of the box in Amazon Managed Grafana.

Currently it's not possible to leverage drag and drop feature in AMG for operational monitoring of AWS IoT SiteWise, as the AWS CloudWatch Namespace for AWS IoT SiteWise is not available in AMG. There is a workaround to this, is by entering the Namespace, Metric Name and Dimensions manually within the AMG. This is quite tideous and difficult to manage. 

Hence, enabling the AWS CloudWatch Namespace for AWS IoT SiteWise within AMG will enable the option to have one central dashboard in AMG, using the out of the box features, instead of having to use both AMG and CloudWatch Monitor.

![AMG](https://user-images.githubusercontent.com/102622291/206330769-4814a6d2-0bca-4010-942a-2ffdaa47cf6f.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
